coders/webp.c

```diff
@@ -249,14 +259,32 @@ static int ReadSingleWEBPImage(Image *image,const uint8_t *stream,
   MagickBooleanType
     status;
 
+  if (is_first) {
+    canvas_width = image->columns;
```

Comment:
Sorry for some more OCD code style comments but could you remove the spaces and change it to `canvas_width=image->columns`? This file already has a mixed convention but I would prefer if we don't introduce new ones. Feel free to reach out if you are in doubt about a convention.